boards/arm/rp2040/common/src/rp2040_common_bringup.c

```diff
@@ -68,6 +68,11 @@
 #include "rp2040_bmp280.h"
 #endif
 
+#ifdef CONFIG_SENSORS_SHT4X
+#include <nuttx/sensors/sht4x.h>
+#include "rp2040_i2c.h"
```

Comment:
You can use rp2040_i2cbus_initialize(busno), it is not board specific. It is arch specific, in fact I think we need some chip_i2cbus_initialize() generic enough to let us move the common/ to the root of boards/

Reply:
The reason I left that function call in the rp2040 bring up is because it seems like the architecture specific logic should stay out of the driver, since I want it to work on all architectures with I2C. I have seen this done for the SHT3x and SHT21.

I was hoping to find some way to access the i2c driver structure without an architecture specific initialization function, but since I haven't been able to, I think this way ensures that the driver can run on all architectures.